### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by luisrpavanelli**
In this PR, we worked, (Me and @MarcoDDM), add modifies the + HTML + CSS, create JS file with the ESLint.

In mobile-menu branch, we implement the features:

- The hamburger button with your open function, the mobile menu appears.
- The close (X) button with your close function, menu disappears.
- The mobile menu options, with you function, the mobile menu disappears.
- The mobile menu options, a correct part of the page is displayed.

Also:
- Modify the README.md
- Create JS file
- License Modified
- Put the Media Screen Sizes

### CODE REVIEW

**Review by didierganthier — COMMENTED** [review]
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't have the JavaScript linters. For this project, you should add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by luisrpavanelli** [comment]
In this PR, we worked, (Me and @MarcoDDM), add modifies the + HTML + CSS, create JS file with the ESLint.

In mobile-menu branch, we implement the features:

The hamburger button with your open function, the mobile menu appears.
The close (X) button with your close function, menu disappears.
The mobile menu options, with you function, the mobile menu disappears.
The mobile menu options, a correct part of the page is displayed.
Also:

Modify the README.md
Create JS file
License Modified
Put the Media Screen Sizes

File: index.html [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
-            <img id="menuicon" src="./Desktop_Icons/menuhb.png" atl="Menu" />
+    <!-- NavBar -->
+    <nav class="navbar">
+        <div class="branding">Welcome</div>
+        <a href="#" class="toggle-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </a>
+        <div class="menu-links">
```
Comment (NahnahAJ) [inline index.html:23]: - [ ] Great job so far :muscle: However, according to the Project Requirements, when you click on the hamburger menu button and the menu opens up, the hamburger icons should disappear leaving the `close button [x]`. However, in your project, the `hamburger icon` and the `close button` were both present when the hamburger was clicked. 
- Please ensure that only the close button is displayed when the menu is open :wink: 
- Your Design
image

- The Figma Design 
image

File: index.html [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
             <ul class="menutabs">
-                <a href="#"><li><a id="textform2">Portfolio</a></li></a>
-                <a href="#"><li><a id="textform2">About</a></li></a>
-                <a href="#"><li><a id="textform2">Contact</a></li></a>
+                <a href="#"><li>Portfolio</li></a>
+                <a href="#"><li>About</li></a>
+                <a href="#"><li>Contact</li></a>
             </ul>
```
Comment (NahnahAJ) [inline index.html:28]: - [ ] According to the Project requirements, 
image
- But I noticed that in your project, whenever I clicked on any of the menu options, It didn't take me to its corresponding section :wink: So kindly fix this  so you can satisfy all the project requirements and increase user interactivity :100: 
- I have attached a video below for your reference :+1: 
[Screencast from 03-07-2023 04:28:39 PM.webm](https://user-images.githubusercontent.com/102699958/223487010-265124c3-591b-4d5a-bbe1-d0a2216f3d8d.webm)

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @MarcoDDM  and @luisrpavanelli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Javascript linters well installed and passing :heavy_check_mark: 
- Hamburger menu added successfully :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
In this PR, we worked, (Me and @MarcoDDM), add modifies the + HTML + CSS, create JS file with the ESLint.

In mobile-menu branch, we implement the features:

The hamburger button with your open function, the mobile menu appears.
The close (X) button with your close function, menu disappears.
The mobile menu options, with you function, the mobile menu disappears.
The mobile menu options, a correct part of the page is displayed.
Also:

Modify the README.md
Create JS file
License Modified
Put the Media Screen Sizes

- Optional suggestions: It's Done

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @luisrpavanelli ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

you-are-near

## Highlights ✔️

- [x] Nice design 
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by luisrpavanelli** [comment]
In this PR, we worked, (Me and @MarcoDDM), add modifies the + HTML + CSS, create JS file with the ESLint.

In mobile-menu branch, we implement the features:

The hamburger button with your open function, the mobile menu appears.
The close (X) button with your close function, menu disappears.
The mobile menu options, with you function, the mobile menu disappears.
The mobile menu options, a correct part of the page is displayed.
Also:

Modify the README.md
Create JS file
License Modified
Put the Media Screen Sizes

Optional suggestions: Every suggest is Done.

 When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed: It Done too

**Review by topeogunleye — COMMENTED** [review]
Hi Team 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a wrong PR from `Portfolio: mobile menu` project to be reviewed as Activity name: `Portfolio: details popup window`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by DuaneDave — APPROVED** [review]
Hi @luisrpavanelli 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Each link are working as expected
✔️ PR summary is very descriptive
✔️ Linters are passing


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**